DirectProgramming/DPC++/DenseLinearAlgebra/simple-add/src/simple-add-usm.cpp

```diff
@@ -63,19 +63,19 @@ int main() {
   // Create device selector for the device of your interest.
 #if FPGA_EMULATOR
   // DPC++ extension: FPGA emulator selector on systems without FPGA card.
```

Comment:
@KanclerzPiotr Do you want to remove the phrase "DPC++ extension" from the comment?

Reply:
I think we should use Intel or oneAPI extension. This is not part of the specification.

Reply:
I fixed this with rename to Intel extension